lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -474,7 +484,10 @@ void MayaHydraSceneIndex::HandleCompleteViewportScene(const MDataServerOperation
         MayaHydraRenderItemAdapterPtr ria = nullptr;
         if (!_GetRenderItem(fastId, ria)) {
             const SdfPath slowId = _GetRenderItemPrimPath(ri);
-            if (slowId.IsEmpty()) {
+
+            // Maya/MtoA adds texturedSkyDome mesh object for VP2.
+            // We do not want that to be translated to Hydra
```

Comment:
For Arnold, we will use mtoa, that is true, but what about HdStorm ? We still want to use the skydeom under storm right ?

Reply:
We do not need the skydome mesh for Storm or Arnold. Both have their own representation. The mesh render item was possibly created for VP2 display, which we do not need.